app/templates/events/manageVolunteers.html

```diff
@@ -119,7 +119,7 @@ <h2 class="accordion-header">
                             {% if participant.hoursEarned == None %}
                               value=""
                             {% else %}
-                              value="{{participant.hoursEarned}}"
+                              value={{participant.hoursEarned}}
```

Comment:
While it's true you might not need to wrap the value in quotes in this particular case, the value should be wrapped in quotes in HTML to match other places on the same page, such as on line 99.